parseComCode/main.js

```diff
@@ -30,6 +30,141 @@ Parse.Cloud.job("populateSearch", function(request, status) {
   });
 });
 
+//A background job to populate usageCounts for languages and tags
+Parse.Cloud.job("populateCounts", function(request, status) {
+    Parse.Cloud.useMasterKey();
+
+    var counters = { language: {}, tag: {}};
+
+    //Query each tag
+    var tagQuery = new Parse.Query('tag');
+    tagQuery.each(function(tag) {
```

Comment:
Is there anything that could imaginably go wrong? Seems like parse queries can take another function that is activated when there's a problem, and maybe we should implement it at least to the extent of aborting everything with a message? Or maybe errors will automatically propagate if not handled?